src/client/components/TopicGrid.tsx

```diff
@@ -2,60 +2,61 @@ import * as React from "react";
 import { useState } from "react";
 import { DataGrid, GridToolbar, GridColDef } from "@mui/x-data-grid";
 import Title from "./Title";
-import { BROKER_METRICS_QUERY } from "../models/queries";
+import { TOPIC_DATAGRID_QUERY } from "../models/queries";
 import { useQuery } from "@apollo/client";
 
 // onQueryCallback with use query
 
+// data grid schema
 const columns: GridColDef[] = [
   { field: "id", headerName: "ID", width: 50 },
   { field: "topic", headerName: "Topic", width: 150 },
   {
     field: "partitionNum",
     headerName: "Partition",
     type: "number",
-    width: 100,
+    width: 150,
   },
   {
     field: "partitionRep",
     headerName: "Replicas per partition",
     type: "number",
-    width: 100,
+    width: 150,
   },
   {
-    field: "isrPerPart",
-    headerName: "# ISRs per partition",
+    field: "underMinISR",
+    headerName: "Under Min ISR",
     type: "number",
-    width: 90,
+    width: 150,
   },
   {
     field: "brokersRep",
     headerName: "# Brokers with replicas",
     type: "number",
-    width: 90,
+    width: 150,
   },
   { field: "logSize", headerName: "logSize", type: "number", width: 90 },
 ];
 
 interface TopicGridProps {
   title?: string;
-  rowCount: number;
+  rowCount?: number;
```

Comment:
This prop needs to be required or the grid doesn't render correctly.

Reply:
Resolved